modules/AWS/eks/variables.tf

```diff
@@ -7,6 +7,15 @@ variable "cluster_name" {
   }
 }
 
+variable "eks_version" {
+  description   = "EKS K8s version"
+  type          = number
+  validation {
+    condition     = can(regex("^1\\.2[1-4]", var.eks_version))
```

Comment:
So we're enforcing only these versions `1.21, 1.22, 1.23, 1.24` because anything else is not tested?

Reply:
Yes, EKS 1.21-1.24 is what's currently supported when you create [a new EKS cluster](https://docs.aws.amazon.com/eks/latest/userguide/kubernetes-versions.html).